tests/test_catalog.py

```diff
@@ -27,6 +28,8 @@
 }
 POINT_BBOX = (1.0, 2.0, 1.0, 2.0)
 
+Geometry = catalog.Geometry
```

Comment:
This one is due to a pylance issue, which does not allow to use imported variables in the test parameters typings.